src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -1109,13 +1109,18 @@ namespace Google.Example.Library.V1
         public virtual Task<Shelf> GetShelfAsync(
             string name,
             SomeMessage message,
-            CallSettings callSettings = null) => GetShelfAsync(
-                new GetShelfRequest
-                {
-                    Name = name,
-                    Message = message,
-                },
-                callSettings);
+            CallSettings callSettings = null)
+        {
+            GetShelfRequest request = new GetShelfRequest
+            {
+                Name = name,
+            };
+            if (message != null)
```

Comment:
For reference types other than string, it's fine to just assign unconditionally; a null reference is just "this field is unset".